lib/hooks/bitbucket/templates/issue_comment_created.html.haml

```diff
@@ -0,0 +1,16 @@
+:ruby
+  issue   = payload.issue
+  actor   = payload.actor
+  comment = payload.comment
+
+%p
+  %span
+    = avatar_image_tag(actor.links.avatar.href)
+
+  %a{href: actor.links.html.href}
+    = actor.display_name
+  created issue comment to
+  %a{href: comment.links.html.href}
+    #{comment.id}
+
+= md comment.content.html
```

Comment:
As Bitbucket's emoji has no size information, this line should be surrounded by `.bitbucket-html` like [this](https://github.com/idobata/idobata-hooks/blob/725465bc0f721d4a02e29e71f43f6fb5d4a99125/lib/hooks/bitbucket/templates/pullrequest_comment_created.html.haml#L13).
Also, I suppose this string isn't Markdown but HTML. Would you like to use `!=` instead of `=`?